### DISCUSSION TITLE
Does Pelican allow to build custom html pages beside the strict structure ?

### DISCUSSION THREAD

**Discussion by 4l1fe** [getpelican/pelican#3291] [Q&A] [answered]
Can [the structure](https://docs.getpelican.com/en/latest/themes.html#structure) be customizable for the case when i have a lot Markdown files, but would like to add a few raw html files with some commercial information that i don't want to write in markdown?

**Comment by Epy** [comment, marked as answer]
You can include some "static" html pages anywhere in your templates, if I got your question correctly:
```
      {% include 'page_infos.html' %}
```
(in my case this one is not pure html, its another Jinja template but it's the same)

**Reply by 4l1fe** [reply]
Yeah, good idea thank you
